### DISCUSSION TITLE
no pick&place(centroid) file

### DISCUSSION THREAD

**Discussion by quelo1972** [frntc/RAD#56] [General]
Hi,
I ordered via PCBWay the rad expansion with assembly service, but they wrote to me the following:
"Dear Customer, Thank you for your PCBA order [T-P7W675700A] , we checked all the files you uploaded and find there is no pick&place(centroid) file. Please go to 'Production Status' to upload this file . If you really can't provide this file, please contact your service representative to explain that."
Can you provide me this "pick&place(centroid)" file Please? Otherwise they can't terminate the fabrication, Thanks.

**Comment by frntc** [comment]
Hey,

please have a look here: https://github.com/frntc/RAD/discussions/43

**Comment by quelo1972** [comment]
I apologize for having inserted a post that had already been discussed in
previous conversations. Then I also saw that post, I wrote to PCBWay and
they replied that they will try to get the centroid file and if they do not
succeed they will proceed with manual assembly even without the file. Thank you very much for the replay.

Il Sab 7 Set 2024, 16:47 frntc ***@***.***> ha scritto:

**Comment by frntc** [comment]
No problem!
